code/web/src/modules/user/api/actions.js

```diff
@@ -117,3 +116,25 @@ export function getGenders() {
     }))
   }
 }
+
+// Updating the user info 
+export function updateUser(user, id) {
+  return dispatch => {
+    dispatch({
+      type: UPDATE_IMAGE,
+      user
+    })
+    return axios.post(routeApi, {
+      query: `
+      mutation userUpdate($image: String!) {
+        userUpdate(id: ${id}, image: $image) {
+          image
+        }
+      }
+    `,
+    variables: {
+      image: user.imgURL,
+    },
+    })
+  }
```

Comment:
Great work here! I had a thought about how to address our issue where we aren't getting an automatic re-render of our component...

I wonder if we can add a `getUser` action, thats responsible for fetching an entire user object from our database. I think it'd look something like this:

`// Get user
export function getUser() {
  return dispatch => {
    return axios.post(routeApi, query({
      operation: 'user',
      fields: ['id', 'name', 'email', 'image', 'role', /* add rest of user fields here*/]
    }))
  }
}`